mobile/src/main/res/values/strings.xml

```diff
@@ -80,19 +82,18 @@
     <string name="error_certificate_expired">SSL certificate has been expired. Please get a new certificate or disable certificate validation during SSL handshake</string>
     <string name="error_certificate_not_valid_yet">SSL certificate is not valid yet. Please get a correct certificate or disable certificate validation during SSL handshake. Also check if the clock on your device is set correctly</string>
     <string name="error_certificate_revoked">SSL certificate has been revoked. Please get a new certificate or disable certificate validation during SSL handshake</string>
-    <string name="error_http_code_401">User authentication has failed. Please check username, password and SSL client certificate (HTTP response code 401)</string>
-    <string name="error_http_code_403">User authentication has failed. Please check username, password and SSL client certificate (HTTP response code 403)</string>
-    <string name="error_http_code_407">User authentication has failed. Please check username, password and SSL client certificate (HTTP response code 407)</string>
-    <string name="error_http_code_414">openHAB URL is too long (HTTP response code 414)</string>
+    <string name="error_http_code_401">Authentication failed. Please check the configured username and password respectively the provided SSL client certificate (HTTP response code 401)</string>
```

Comment:
Should I add a full stop at the end of the message?

Reply:
Whoever decides that also has to go through many of the other strings...